configs/db_demo.yaml

```diff
@@ -148,7 +148,21 @@ sources:
       sub: skyportal/tests/data/16fil_sub.png
     token: "load_demo_data token"
 taxonomies:
-  - name: Demo taxonomy
+  - name: Simple Taxonomy
```

Comment:
New taxonomy shows that taxonomy select list is not ordred by Taxonomy ID. Consider sorting this list by ID, or removing the ID from the list

<img width="601" alt="Screen Shot 2020-07-23 at 8 20 31 PM" src="https://user-images.githubusercontent.com/2769632/88357992-231e6c80-cd22-11ea-8354-2302478b713a.png">

Reply:
Done.